client/src/services/sorting.js

```diff
@@ -15,4 +15,9 @@ const sortByCode = (a, b) => {
   return 0;
 };
 
-export default { sortByCode };
+const sortByDate = (a, b) => {
```

Comment:
`compareDate` might be a better name since this function doesn't do any sorting by itself. A comment explaining that this is in descending order could also be good.

Reply:
I kept the name to match the naming of the function above

Reply:
That could potentially also be renamed to `compareCode`, but it's not a huge deal, consistency is of course always important and I don't want to delay merging this for too long.